src/ui/marker.js

```diff
@@ -484,4 +510,61 @@ export default class Marker extends Evented {
     isDraggable() {
         return this._draggable;
     }
+
+    /**
+     * Sets the `rotation` property for pointing the marker in a particular direction
```

Comment:
I think the wording on this comment isn't quite right. If the marker is set relative to the viewport, then it's not really correct to say it's pointing in a particular "direction".  There's also no indication in the description of the method or the `rotation` parameter that `rotation` is in degrees or what positive and negative degrees would do (clockwise vs counter-clockwise).

Reply:
Agreed this should be changed, but it goes back to if we should repeat the documentation for the property, or omit duplicating it and expect people to refer back to the `rotation` property documentation.

I think less duplication is better as you don't need to re-read the same documentation in multiple places and ensure it's consistent.

` * @param {number} [options.rotation=0] The rotation angle of the marker (clockwise, in degrees), relative to its respective {@link Marker#rotationAlignment} setting.`

Reply:
Updated to "Sets the `rotationAlignment` of the marker."